Update README.md (#112)
mattermost/mattermost-handbook: operations/workplace/README.md

# Workplace

> ## What is the Mattermost Community Server?

Reviewer: This section is actually quite outdated. Today, we have three separate "community" servers:

- [community.mattermost.com](https://community.mattermost.com) running our last stable release
- [community-release.mattermost.com](https://community-release.mattermost.com) running our upcoming stable release)
- [community-daily.mattermost.com](https://community-daily.mattermost.com) running the latest builds of Mattermost's software

Reviewer: We could update the text to say

```
## What is the Mattermost Community Server?

The Mattermost community server runs the latest stable build of Mattermost's software, including all open source code for Mattermost Team Edition as well as commercial code for Mattermost Enterprise Edition.

It has two objectives:

* Provide a workplace for Mattermost staff.
* Engage with our [communities ](../../contributors/contributors/community.md), including users, contributors and partners.

**The Mattermost community server is not a production server and has occasional downtime.**

You can reach the server at:

{% embed url="https://community.mattermost.com" caption="" %}

The server is also available via the following URLs, primarily used to test pre-release features:

 - [https://community-release.mattermost.com](https://community-release.mattermost.com), which runs our upcoming stable release. This server includes work-in-progress features, will contain bugs and have occasional downtime. This server receives regular pushes of features and/or bug fixes of the next release for testing purposes
 - [https://community-daily.mattermost.com](https://community-daily.mattermost.com), which runs latest builds of Mattermost's software, including all pre-release features of our core products. It is the most unstable version of the Mattermost community server. Bugs and occasional downtime is expected as this server receives daily pushes of features and/or bug fixes for testing purposes.
```

Author: Thanks Jason, I've updated it.

> ## What is the Mattermost Pre-release Server?

Reviewer 2: I'm not 100% sure if it makes sense to call it pre-release server since it is now called community server. Not sure who would know what the norm these days is for the name of the server.

Author: I meant to comment on that and I agree. Let's go with community seeing as that's the link name, and what we commonly refer to?

> ## What is the Mattermost Pre-release Server?

Author: ```suggestion
## What is the Mattermost Community Server?
```

The Mattermost pre-release server runs the latest builds of Mattermost's software, including all open source code for Mattermost Team Edition as well as commercial code for Mattermost Enterprise Edition.

It has three objectives:

* Provide a workplace for Mattermost staff.

> * Test pre-released builds of our core products.

Author: ```suggestion
* Test work-in-progress builds of our core products.
```

> * Test work-in-progress builds of our core products.

Reviewer 2: ```suggestion
* Test pre-released builds of our core products.
```

In this case I believe it's fine to say "pre-released" as this is not referring to the server's name.

* Engage with our [communities ](../../contributors/contributors/community.md)including contributors and partners.

> **The Mattermost pre-release server is NOT A PRODUCTION SERVER.** It is a live view of work-in-progress and will contain bugs and have occasional downtime.

Author: ```suggestion
**The Mattermost Community server is NOT A PRODUCTION SERVER.** It is a live view of work-in-progress and will contain bugs and have occasional downtime.
```

You can reach the server at:

{% embed url="https://community.mattermost.com" caption="" %}

> The server is also available via the URL [https://community.mattermost.com](https://community.mattermost.com) which receives the same regular pushes of partially tested code as [https://community-daily.mattermost.com](https://community-daily.mattermost.com) which offers daily builds.

Reviewer 2: Since community server is already mentioned above, we can probably change the first link to ``https://community-release.mattermost.com`` and mention that it receives regular pushes of features and/or bug fixes of the next release for testing purposes.

Author: Awesome, thanks!

The server has two teams:

### **Contributors Team**

URL: [https://community.mattermost.com/core](https://community.mattermost.com/core/messages/@github)

> Available to anyone who'd like to join the server using either an email and password account \([you can sign-up for a free account online](https://pre-release.mattermost.com/signup_user_complete)\) or a Mattermost staff account, which is set up via OneLogin and requires MFA either through OneLogin or Google Authenticator.

Reviewer 2: ```suggestion
Available to anyone who'd like to join the server using either an email and password account \([you can sign-up for a free account online](https://community.mattermost.com/signup_user_complete)\) or a Mattermost staff account, which is set up via OneLogin and requires MFA either through OneLogin or Google Authenticator.
```

> ### **Staff Team**

Reviewer: There's a formatting error at the end of the second sentence here (`... MFA**.**`)

![image](https://user-images.githubusercontent.com/13119842/78721537-beb3da80-78f5-11ea-9352-3e2b22b055d4.png)

# Workplace

## What is the Mattermost community server?

What is the Mattermost Community Server?

The Mattermost community server runs the latest stable build of Mattermost's software, including all open source code for Mattermost Team Edition as well as commercial code for Mattermost Enterprise Edition.

It has two objectives:

* Provide a workplace for Mattermost staff.

> * Engage with our [communities ](../../contributors/contributors/community.md), including users, contributors and partners.

Reviewer 2: ```suggestion
* Engage with our [communities](../../contributors/contributors/community.md), including users, contributors and partners.
```

**The Mattermost community server is not a production server and has occasional downtime.**

You can reach the server at:

{% embed url="https://community.mattermost.com" caption="" %}

The server is also available via the following URLs, primarily used to test pre-release features:

>  - [https://community-release.mattermost.com](https://community-release.mattermost.com), which runs our upcoming stable release. This server includes work-in-progress features, will contain bugs and have occasional downtime. This server receives regular pushes of features and/or bug fixes of the next release for testing purposes.

Reviewer 2: ```suggestion
 - [https://community-release.mattermost.com](https://community-release.mattermost.com), which runs our upcoming stable release. This server includes work-in-progress features, and can contain bugs and have occasional downtime. This server receives regular pushes of features and/or bug fixes of the next release for testing purposes.
```

URL: [https://community.mattermost.com/core](https://community.mattermost.com/core/messages/@github)

> Available to anyone who'd like to join the server using either an email and password account \([you can sign-up for a free account online](https://pre-release.mattermost.com/signup_user_complete)\) or a Mattermost staff account, which is set up via OneLogin and requires MFA either through OneLogin or Google Authenticator.

Reviewer 2: Not strictly part of this PR but the ``pre-release.mattermost.com`` is an old server so not sure we want to link to it throughout this README page.
Also, ``https://daily-community.mattermost.com`` should be ``https://community-daily.mattermost.com``.

Author: Thanks, @amyblais, fixed the community-daily one. 

I'm happy to remove the link to the pre-release server. Do we have a replacement, or can I adjust that sentence to include the other two servers only?

Reviewer 2: The replacement is https://community.mattermost.com.

By default, people's user names and first names trigger mention notifications in Mattermost. This can be distracting if people have the same first names, for example there are multiple people named "Chris" at Mattermost. If you want to mention someone without triggering a notification consider using their three letter initials, for example, "Alice Emily Evans" would be `AEE`.

As an alternative to saying `@alice.evans` using `AEE` lets you refer to the person in passing without pushing an alert to them. For staff members that want to be alerted to even passing mentions of their name, they can opt in to that model by creating a [keyword notification](https://docs.mattermost.com/help/messaging/mentioning-teammates.html#words-that-trigger-mentions) on their three letter initials.

## Key Channels for Staff

The following table lists popular channels for Staff:

> | Purpose | Name | Explanation |

Reviewer: ```suggestion
| Purpose | Name | Description |
```
